src/main/flight/altitude.c

```diff
@@ -231,6 +233,12 @@ void calculateEstimatedAltitude(timeUs_t currentTimeUs)
             estimatedAltitude = baroAlt;
         }
     }
+#else
```

Comment:
This will not work for a lot of targets - `USE_BARO` is defined in a lot of cases to support external BARO sensors, but in most cases no BARO will be detected.